packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeActionButtons.tsx

```diff
@@ -9,7 +9,7 @@ import { HierarchyLevelDetails, useTree } from "../UseTree.js";
 import { useLocalizationContext } from "./LocalizationContext.js";
 
 const filterIcon = new URL("@itwin/itwinui-icons/filter.svg", import.meta.url).href;
-const dismissIcon = new URL("@itwin/itwinui-icons/dismiss.svg", import.meta.url).href;
+const activeFilterIcon = new URL("@itwin/itwinui-icons/placeholder.svg", import.meta.url).href; // Placeholder
```

Comment:
does this mean we're missing an icon? if yes, please add it to "Icons status" list in https://github.com/iTwin/viewer-components-react/issues/1149#issue-2808932282

Reply:
I was under the impression that this includes it.
![image](https://github.com/user-attachments/assets/6ac7fede-63ee-4999-8691-98c7a8c62ce7)

Reply:
yes, my bad (somehow I thought we need two icons - regular one and "active filter" one)